packages/react/helmet/react-helmet-shim.js

```diff
@@ -0,0 +1,10 @@
+const deprecate = require('depd')('hops-react');
+const reactHelmetAsync = require('react-helmet-async');
+
+deprecate(
+  '[DEP004] react-helmet is deprecated. Please switch to react-helmet-async instead (https://github.com/untool/untool/blob/master/DEPRECATIONS.md#dep004).'
```

Comment:
there is already a `[DEP004]` & also the URL to the `DEPRECATIONS.md` is wrong.

Reply:
We decided to keep the untool deprecations in place and have them link to untool, because otherwise we would need to change almost all deprecation numbers and urls.